### PR TITLE
Enable SSCS

### PR DESCRIPTION
**Pull request by doncem**
### Change description ###

Onboard SSCS team

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -63,6 +63,8 @@ ftp:
       folder: DIVORCE
     - service: probate_backend
       folder: PROBATE
+    - service: sscs
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml]: please check the s2s service name

Comment (ryandac) [reply src/main/resources/application.yaml]: @jackmaloney do you know what an s2s service name is?

Comment (doncem) [reply src/main/resources/application.yaml]: logs suggested this was the service accessing send letter

Comment (ryandac) [reply src/main/resources/application.yaml]: I'm reliably informed it's set here https://github.com/hmcts/sscs-evidence-share/blob/master/src/main/resources/application.yaml#L54 and is usually sscs

**Pull request merged**